lib/ssh_scan/database/mongo_db.rb

```diff
@@ -0,0 +1,40 @@
+require 'mongo'
```

Comment:
The naming on these files is redundant.  I would consider removing the "_db" as you're already name spaced as a datbase in the file name and in the Ruby module namespace.